.eslintignore

```diff
@@ -0,0 +1,2 @@
+playground
+legacy
```

Comment:
Could we move `.eslint` files to a separate PR. They're great, but I want to keep this PR as focused as possible.

Additionally can we have trailing new lines, please.